runtime/src/disable.rs

```diff
@@ -0,0 +1,56 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{Drivers, MailboxResp};
+use caliptra_common::keyids::{KEY_ID_RT_CDI, KEY_ID_RT_PRIV_KEY, KEY_ID_TMP};
+use caliptra_drivers::{
+    hmac384_kdf, Array4x12, CaliptraError, CaliptraResult, Ecc384PrivKeyOut, KeyReadArgs, KeyUsage,
+    KeyWriteArgs,
+};
+
+pub struct DisableAttestationCmd;
+impl DisableAttestationCmd {
+    pub(crate) fn execute(drivers: &mut Drivers) -> CaliptraResult<MailboxResp> {
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_CDI)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_PRIV_KEY)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+        Self::generate_dice_key(drivers)?;
+        Ok(MailboxResp::default())
+    }
+
+    fn generate_dice_key(drivers: &mut Drivers) -> CaliptraResult<()> {
+        hmac384_kdf(
+            &mut drivers.hmac384,
+            KeyReadArgs::new(KEY_ID_RT_CDI).into(),
```

Comment:
I'm not completely sure what happens  when you erase a keyvault slot and then use it uninitialized. The SW emulator may allow it, but I'm not sure the real hardware will.

Could you make a rom test and then run it against the verilator model to be sure? Ideally we would explicitly fill the CDI keyvault slot with zeros, but I'm not sure if there's a way to do that.

A few tips on verilator:

* Instructions for using verilator: https://github.com/chipsalliance/caliptra-sw#testing-against-verilator. You'll need to install it and the build will take a while
* Verilator is pretty slow, so the test may take longer than expected to run
* You can use the `test_ecdsa_verify_cmd` test as an example of something that can run against verilator but still run mailbox commands